.travis.yml

```diff
@@ -3,7 +3,7 @@
 # run xvfb with 32-bit color
 # xvfb-run -s '-screen 0 1600x1200x24+32' command_goes_here
 
-matrix:
+jobs:
```

Comment:
Just curious, why change this?

Reply:
Travis now uses jobs instead of matrix and their docs have all changed to remove matrix. https://docs.travis-ci.com/user/build-matrix/